docs/usage/using-wp-mock.md

```diff
@@ -55,6 +54,17 @@ Calling `WP_Mock::userFunction()` will dynamically define the function for you i
 return \WP_Mock\Functions\Handler::handleFunction(__FUNCTION__, func_get_args());
 ```
 
+
+{% hint style="warning" %}
+
+**Important!**
+
+Note how in the above test class `WP_Mock\Tools\TestCase` was extended instead of `PHPUnit\Framework\TestCase`. This is because WP_Mock provides a custom test case class that extends PHPUnit's test case class and adds additional functionality to it. Ideally, you should always extend `WP_Mock\Tools\TestCase` instead of `PHPUnit\Framework\TestCase` when using WP_Mock. _If you do not wish to extend WP_Mock own test case class_ you should make sure to call `WP_Mock::setUp()` and `WP_Mock::tearDown()` in your test case's `setUp()` and `tearDown()` methods respectively.
```

Comment:
I kinda feel like some or all of this should probably live in the Configuration page as opposed to here. That's not to say it wouldn't be helpful to reinforce that information here, I'm just thinking that if we're placing high importance on this information, it really is a Configuration issue.

Reply:
And to be fair, WP_Mock's `TestCase` does specifically make `userFunction()` more flexible in the context of this kind of usage, I just feel like we're sending mixed message throughout these docs that could be better aligned by recommending `TestCase` from the beginning. 

So instead of the vibe being "WP_Mock's `TestCase` is totally optional, but it comes with some really great helper methods" I'm suggesting we move more toward "We highly recommend extending WP_Mock's `TestCase` for the standard use case. Choosing not to extend it means you will need to implement X, Y, Z processes to cover your bases."

Reply:
fair enough, we can have that in both places

Reply:
see [`48c99e8` (#239)](https://github.com/10up/wp_mock/pull/239/commits/48c99e88476a1d92b4fb2cca333cc40dc729f6ea) where I applied some of your suggestions, @ajaynes-godaddy

Reply:
Looks great @unfulvio-godaddy!